app/v2/connectors/DepartureConnector.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.connectors
+
+import config.AppConfig
+import connectors.util.CustomHttpReader
+import play.api.http.Status.OK
+import v2.models.DepartureId
+import v2.models.DepartureWithoutMessages
+import v2.models.EORINumber
+import v2.models.Message
+import v2.models.MessageId
+import play.api.mvc.RequestHeader
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpClient
+import uk.gov.hmrc.http.HttpResponse
+import utils.Utils
+import v2.models.formats.CommonFormats.departureWithoutMessagesFormat
+import v2.models.formats.CommonFormats.messageFormat
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+
+class DepartureConnector @Inject()(http: HttpClient, appConfig: AppConfig) extends BaseConnector {
+
+  def getDeparture(eori: EORINumber, departureId: DepartureId)(implicit
+                                                               requestHeader: RequestHeader,
+                                                               hc: HeaderCarrier,
+                                                               ec: ExecutionContext): Future[DepartureWithoutMessages] = {
+    val url = s"${appConfig.transitMovementsUrl}/traders/${eori.value}/movements/departures/${departureId.value}"
```

Comment:
```suggestion
    val url = s"${appConfig.transitMovementsUrl}/transit-movements/traders/${eori.value}/movements/departures/${departureId.value}"
```

Reply:
With this, I get a 500, which is expected because the failure is that there is no endpoint on the router yet, I see it being called here: 

```
022-09-12 13:25:32,243 level=[DEBUG] logger=[connector] thread=[scala-execution-context-global-202] rid=[] user=[] message=[:POST:421225881:421.225ms:1110293001:1.110s:96d3:http://localhost:9478/traders/99999912345/movements/631f233aa410d0e2-0000000000000000/messages/:ok] 
```

Reply:
@drnaylor-oc transitMovementsUrl evaluates to transit-movements, so this change would add it again?

Reply:
It's the change I had to make on my local copy to get this working. `appConfig.transitMovementsUrl` appears to evaluate to `http://localhost:9520` only.

Reply:
Yeah you're right, it evaluates to $protocol://$host:$port